cmd/operator/register/root.go

```diff
@@ -293,7 +293,11 @@ func callElementalClient(conf cfg.Elemental, cloudInitPath string) error {
 		return err
 	}
 
-	installerOpts := []string{"elemental", "install"}
+	installerOpts := []string{"elemental"}
+	if conf.Install.Debug {
```

Comment:
So, we would not need anymore to export the env var ELEMENTAL_DEBUG (if it ever worked)